### DISCUSSION TITLE
VTT/SRT directly from API response.

### DISCUSSION THREAD

**Discussion by shaquib97** [deepgram/community#589] [General help] [answered]
Is there a way to convert API response to VTT/SRT format without processing with the SDK? 
I am using it in my Java application, but seems like there is no way to convert the response to VTT in java.

Or can you suggest some library that can help me do so?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by lukeocodes** [comment, marked as answer]
Hi, thanks for your question. We do not format WebVTT or SRT directly in the API response. We have however separated our WebVTT and SRT formatting for use standalone from our SDKs. We don't have a Java version of this right now, but we hoped to produce one in the future.

If you're proficient with Python or TypeScript, you can check out our two open source captions formatting libraries to see how we turn our response into the required formats.

https://github.com/deepgram/deepgram-js-captions
https://github.com/deepgram/deepgram-python-captions
